script/release

```diff
@@ -89,5 +82,11 @@ puts
 
 verbose("git push origin HEAD:release -f")
 
+if release_option != "5"
+  verbose("git push origin HEAD:release -f")
+else
+  verbose("git push -f")
```

Comment:
which branch would this push to?

Reply:
The current branch which is hotfix.

Reply:
Can we explicitly push to hotfix branch? Instead of assuming that the use is on the hotfix branch when they run this script?